build/travis/script.sh

```diff
@@ -7,3 +7,5 @@ php phpunit.php -c ../../extensions/PropertySuggester/phpunit.xml.dist
 
 # cd ../wiki/extensions/PropertySuggester
```

Comment:
maybe we need this line again or something like this to get back to the main extension directory?